algo-builder-tester/test/v6trade-test.ts

```diff
@@ -0,0 +1,401 @@
+import { AccountStore, Runtime, types as typesRT } from "@algo-builder/runtime";
+import { parsing, types as typesW } from "@algo-builder/web";
+
+import { LogicSigAccount } from "algosdk";
+import { assert } from "chai";
+
+// TEST PARAMS
+const runtime_config = {
+  approvalProgramFileName: "Bitcoin-approval.teal",
```

Comment:
algob support put pyteal file directly

Reply:
I know.. this is before you answered the `algob deploy`. The whole `algob-tester.py` (also `price-verifier.py` in #8) was written in this way.